src/Akka.Persistence.Sql.Linq2Db/Journal/DAO/BaseByteArrayJournalDao.cs

```diff
@@ -335,6 +337,13 @@ await db.GetTable<JournalMetaData>()
                         .DeleteAsync();
                 }
 
+                if (JournalConfig.TableConfig.TagWriteMode != TagWriteMode.Csv)
+                {
+                    await db.GetTable<JournalTagRow>()
+                        .Where(r => r.SequenceNumber <= maxSequenceNr && r.PersistenceId == persistenceId)
+                        .DeleteAsync();
```

Comment:
TagTable row deletion, simplified because we don't have to cache the ordering numbers that has been deleted

Reply:
LGTM